### PR TITLE
Reactify declaration incomes page

### PR DESCRIPTION
**Pull request by frederike-ramin**
# Short Description
- More React! :)

# Changes
- Add React component
- Wire it up

# Feedback
- Do you think we need a test that the list is displayed?
- Is there another way of accessing an image than loading it in the /assets folder?

### CODE REVIEW

File: webapp/client/src/assets/images/list_marker.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="27" height="16" viewBox="0 -10 27 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (nfelger) [inline webapp/client/src/assets/images/list_marker.svg:1]: I think this should go into `assets/icons` not `assets/images`. No need to replicate the mistakes of old… :D

**Review by nfelger — CHANGES_REQUESTED** [review]
> Do you think we need a test that the list is displayed?

I don't think the page component needs a test, there's no logic involved in showing the list. I _do_ think the page needs a functional test though, to make sure the wiring between Python and React is correct. Could you add that please?

> Is there another way of accessing an image than loading it in the /assets folder?

What do you have in mind? Or rather: what doesn't work for you about loading it from the `/assets` folder?

**Comment by frederike-ramin** [comment]
> What do you have in mind? Or rather: what doesn't work for you about loading it from the /assets folder?

Just because I'm duplicating the icon - that's the only reason I asked I think.

**Comment by nfelger** [comment]
> Just because I'm duplicating the icon - that's the only reason I asked I think.

Right. When we're done, we can delete all assets from public/ except what's used by the Lay-out templates and non-react pages. I'll make a "post-reactification cleanup" ticket where we can collect tasks like this and also removing CSS classes from components.

**Comment by nfelger** [comment]
Feel free to add anything I missed: https://steuerlotse.atlassian.net/browse/STL-1575

**Review by nfelger — APPROVED** [review]

**Review by SannyNguyenHung — APPROVED** [review]
`Do you think we need a test that the list is displayed?`

As Nico already said, there is no functionality behind it. So we don't need to test this.

`Is there another way of accessing an image than loading it in the /assets folder?`

Already discussed with Nico :)

PR is good :+1: let's merge it

**Pull request merged**